src/lib/orionTypes/EntityTypesResponse.cpp

```diff
@@ -66,11 +66,6 @@ std::string EntityTypesResponse::render(ConnectionInfo* ciP, const std::string&
   std::string out                 = "";
   std::string tag                 = "entityTypesResponse";
 
-  if ((ciP->uriParam["attributesFormat"] == "object") && (ciP->outFormat == JSON))
```

Comment:
With the new ormat, attributeFormat is no longer needed, so this checking has been removed, as long as the renderAsJsonObject() functions.

NTC (just informative)